content/blog/2020-04-16-april-20-community-gems.md

```diff
@@ -0,0 +1,108 @@
+---
+title: April '20 Community Gems
+date: 2020-04-16
+description: |
+  Great discussions and technical Q&A's from our users.
+descriptionLong: |
+  Look here every month for great discussions and technical Q&A's from our users 
+  and core development team.
+picture: ../../static/uploads/images/2020-04-16/DVC_Gems_April_20.png
+author: ../authors/elle_obrien.md
+commentsUrl: https://discuss.dvc.org/t/march-20-community-gems/336
+tags:
+  - Discord
+---
+
+## Discord gems
+
+Here are some Q&A's from our Discord channel that we think are worth sharing.
+
+### Q: [How can I view and download files that are being tracked by DVC in a repository?](https://discordapp.com/channels/485586884165107732/485596304961962003/698815826870009868)
+
+To list the files that are currently being tracked in a project repository by
+DVC and Git, you can use `dvc list`. This will display the contents of that
+repository, including `.dvc` files. To download the contents corresponding to a
+particular `.dvc` file, use `dvc get`:
+
+Let's consider an example using both functions. Assume we're working with DVC's
+data registry example repository. To list the files present, run:
+
+```dvc
+$ dvc list -R https://github.com/iterative/dataset-registry
+.gitignore
+README.md
+get-started/.gitignore
+get-started/data.xml
+get-started/data.xml.dvc
+...
+```
+
+Note that the `-R` flag, which enables `dvc list` to display the contents of
+directories inside the repository. Now assume you want to download `data.xml`,
+which we can see is being tracked by DVC. To download the dataset to your local
+workspace, you would then run
+
+```dvc
+$ dvc get https://github.com/iterative/dataset-registry get-started/data.xml
+```
+
+For more examples and information,
+[see the documents](https://dvc.org/doc/command-reference/list#list) for
+`dvc list` and for [`dvc get`](https://dvc.org/doc/command-reference/get).
+
+### Q: [I'm setting up an S3 remote and I'd like to forbid `dvc gc --cloud` so users can't accidently delete files in the remote. Will it be sufficient to restrict deletion in S3 settings?](https://discordapp.com/channels/485586884165107732/563406153334128681/698116671298076672)
+
+You're right to be careful, because `dvc gc --cloud` can be dangerous in the
+wrong hands- it'll remove any unused files in your S3 remote (for more info,
+[see our docs](https://dvc.org/doc/command-reference/gc#gc)). To prevent users
+from having this power, setting your S3 bucket policy to block object deletions
+should do the trick (check out this
+[relevant Stack Overflow question](https://stackoverflow.com/questions/49693706/disable-delete-option-for-s3-objects-in-aws)).
+
+### Q: [My team is interested in DVC, and we have all of our data in S3 remote storage. Do we need to install a centralised enterprise version of DVC on a dedicated server? And do we have to also have a GitHub repository?](https://discordapp.com/channels/485586884165107732/563406153334128681/692524884701478992)
```

Comment:
same here, it's not AWS specific question probably if we generalize it? (it's good to mention S3 though for search)

Reply:
good point, done